src/DynamoCoreWpf/ViewModels/PackageManager/Docs/PublishPackageDocumentation.html

```diff
@@ -0,0 +1,226 @@
+<!-- and custom styling in Dynamo sidepanel. -->
+
+<body style="background-color: #353535; font-family: 'Artifakt Element', 'Open Sans';">
+    <h3>Publishing Dynamo Packages</h3>
+    <p>
+        Dynamo offers a vast no. of features out of the box, along with those we also maintain an extensive package library,
+        contributed by the Dynamo community, these can significantly extend Dynamo's capability. This documentation describes how a user can publish a package from Dynamo.
+    </p>
+    <p>
+        The menu option to publish a package can be found at: "Packages > Publish New Package...",
+        the dialog box that opens up after it will ask for all the information related to the package the user wants to publish.
+    </p>
+    <h4>Package Name:</h4>
+    <p>
+        Name of the package you want to publish. This will appear in the library and on the package manager website.
+    </p>
+    <p>
+        <b>Description:</b><br />
+        This is a great place to add more details about your package, you can list any exceptions and pre-requisites.
+        This can be used to educate users on what the package does and how it can be utilised into their workspaces.
+    </p>
+    <p>
+        <b>Version:</b><br />
+        The version field is bifurcated into major/minor/build sections as per the versioning semantics.
+        Users may want to publish more versions for their packages with new and improved features, or just to polish some corners which were left out in the first version.
+    </p>
+    <p>
+        <b>Group:</b><br />
+        This field can be used to specify a group that a package belongs to such as the company/organization.
+    </p>
+    <p>
+        <b>External Dependency (optional):</b><br />
+        Your package may depend on a host, as dynamo can be integrated with a bunch of host options such as Revit, Civil 3D etc.,
+        So this field can be used to specify which other host(s) does your package depends upon.
+    </p>
+    <p>
+        <b>Keywords (optional):</b><br />
+        Some keywords that can be used as tags related to your package. For e.g Generative Design, ML etc.
+    </p>
+    <p>
+        <b>Website (optional):</b><br />
+        Website that may provide the user with more info regarding the package or the organization/author.
+    </p>
+    <p>
+        <b>Repository URL (optional):</b><br />
+        A link to package's repository that can help users to view or contribute to the package development.
+    </p>
+    <p>
+        <b>License (optional):</b><br />
+        A license under which the package is released. Default license : MIT.
+    </p>
+    <p>
+        <b>Package Contents (optional):</b><br />
+        This section includes the right sidebar of the dialog box.
+    </p>
+    <p>
+        <b>Add File:</b><br />
+        Select and add all the files that are related to your package, generally the bin folder, binaries and dependencies are added.
```

Comment:
Suggest adding a colon:

Select and add all the files that are related to your package; generally the bin folder, binaries and dependencies are added.